__tests__/utils/formatDate.test.ts

```diff
@@ -12,27 +12,57 @@ afterEach(() => {
 
 describe('formatDate', () => {
     it('should return 0s ago', () => {
-        const result = formatDate('2021-08-09T17:59:59.000Z');
+        const result = formatDate({
+            inputDate: '2021-08-09T17:59:59.000Z',
+            relativeDuration: true,
+            fullDate: false,
+        });
         expect(result).toEqual('1s ago');
     });
 
     it('should return 1m ago', () => {
-        const result = formatDate('2021-08-09T17:58:59.000Z');
+        const result = formatDate({
+            inputDate: '2021-08-09T17:58:59.000Z',
+            relativeDuration: true,
+            fullDate: false,
+        });
         expect(result).toEqual('1m ago');
     });
 
     it('should return 1h ago', () => {
-        const result = formatDate('2021-08-09T17:00:00.000Z');
+        const result = formatDate({
+            inputDate: '2021-08-09T17:00:00.000Z',
+            relativeDuration: true,
+            fullDate: false,
+        });
         expect(result).toEqual('1h ago');
     });
 
     it('should return 1d ago', () => {
-        const result = formatDate('2021-08-08T18:00:00.000Z');
+        const result = formatDate({
+            inputDate: '2021-08-08T18:00:00.000Z',
+            relativeDuration: true,
+            fullDate: false,
+        });
         expect(result).toEqual('1d ago');
     });
 
-    it('should return Aug 9, 2021', () => {
-        const result = formatDate('2021-08-05T00:00:00.000Z');
-        expect(result).toEqual('Aug 5, 2021');
+    it('should return Aug 5, 2021', () => {
+        const result = formatDate({
+            inputDate: '2021-08-05T00:00:00.000Z',
+            relativeDuration: false,
+            fullDate: false,
+        });
+        expect(result).toEqual('August 5, 2021');
+    });
+
+    // Add more test cases based on the function's behavior
```

Comment:
is this comment still relevant or can be removed ?

Reply:
Removed